docs/topics/scope-functions.md

```diff
@@ -112,7 +112,7 @@ fun main() {
 
     // it
     str.let {
-        println("The receiver string's length is ${it.length}")
+        println("The argument string's length is ${it.length}")
```

Comment:
But have some doubts about this edit.

Reply:
Well, it's still the receiver of `let`, definitely not its argument :)
If you find this contradictory, we can leave just `"The string's length is", but I don't see a problem here.

Reply:
>Well, it's still the receiver of let, definitely not its argument :)
And we don't write that it's the argument of `let`, `it` is a lambda argument (the same as `this` is a lambda receiver above) and we take the length of `it`.
WDYT about it in this perspective?

Reply:
My thoughts are the same. 
- We can remove the word to avoid overloading readers with details. It's obvious what is _the string_ here.
- We can call it receiver because it stands before the dot and the function we talk about. 

Other options seem incorrect to me.

Reply:
Removed the word "string".